packages/desktop-gui/src/specs/specs-store.js

```diff
@@ -77,6 +83,10 @@ export class SpecsStore {
       })
     }))
 
+    if (this.newSpecAbsolutePath && !_.find(this._files, { absolute: this.newSpecAbsolutePath })) {
```

Comment:
yeah, i bet the paths in this._files are either double-escaped Windows paths, or POSIX paths, that's why this is not working as expected on Windows